src/ripple/core/tests/Config.test.cpp

```diff
@@ -138,28 +128,26 @@ backend=sqlite
 class ConfigGuard
 {
 private:
+    bool rmSubDir_{false};
+
+protected:
     using path = boost::filesystem::path;
     path subDir_;
-    path configFile_;
-    path dataDir_;
 
-    bool rmSubDir_{false};
-    bool rmDataDir_{false};
-
-    Config config_;
+    auto rmDir (path const& toRm)
+    {
+        if (is_directory (toRm) && is_empty (toRm))
+            remove (toRm);
+        else
+            std::cerr << "Expected " << toRm.string ()
+                      << " to be an empty existing directory.\n";
```

Comment:
Soon, `beast::unit_test::suite::log` will be derived from `std::ostream` so you can pass the log to functions which want to log.